library/src/ui/index.js

```diff
@@ -85,4 +85,59 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  DisplayComponent: (function () {
+    function constructor() {}
+
+    constructor.prototype.setDeltaXY = function ({ x, y }) {
+      common.deltaXY = { x: x, y: y };
+    };
+
+    constructor.prototype.prepareData = function (tablesJSON) {
+      var tablesList = [];
+      for (let i = 0; i < tablesJSON.length; i++) {
+        var { name, classType, coordinates, columns, rows } = tablesJSON[i];
+        var rowsPrepared = [];
+        for (let j = 0; j < columns.conditions.length; j++) {
+          rowsPrepared.push(rows.conditions[i][1] || "");
+        }
+        for (let j = 0; j < columns.decisions.length; j++) {
+          rowsPrepared.push(rows.decisions[i][1] || "");
+        }
+
+        tablesList.push({
+          name: name,
+          class: classType,
+          coordinates: coordinates,
+          conditions: columns.conditions,
+          decisions: columns.decisions,
+          rows: rowsPrepared,
+        });
+      }
+
+      return tablesList;
+    };
+
+    constructor.prototype.onInit = function (common, props) {
+      this.children = [];
```

Comment:
A taka inicjalizacja nie istnieje w base component? Jak nie to trzeba przenieść, bo to się będzie wszędzie powtarzało.